analyses/snv-callers/compare_snv_callers_plots.Rmd

```diff
@@ -12,28 +12,30 @@ Purpose: After creating a consensus file with `02-merge_callers.R`, this noteboo
 plots these data in a series of plots to compare the consensus calls to the individual 
 callers. 
 
-### Summary of Findings:
-
-
-#### Individual Caller's Reports 
-
-
 *Notable Findings:*  
 
 - [VarDict calls more mutations](#upset-graph) than any of the other callers. 
 Many of these have [very low ](#vaf-distributions) suggesting that some may be sequencing errors. 
 
 - Mutect2, Strelka2, and Lancet do have [plenty of mutations in common](#upset-graph). 
```

Comment:
I feel like we can promote a bit of info toward the top here. Of course, it might not be accurate if the analysis changes, but I feel like it is worth making a point of
```suggestion
- Mutect2, Strelka2, and Lancet do have [a large number of mutations in common](#upset-graphs). 
- A 2 out of 3 consensus would increase the number of mutations called ~4X, but that set would be almost equivalent to a Mutect2 & Strelka2 consensus.
  -There are relatively few mutations found in only Mutect & Lancet or Strelka & Lancet (~125,000 vs 5,600,000 total mutations in at least 2 of 3 callers)
```